clang/test/Analysis/eval-predefined-exprs.cpp

```diff
@@ -52,12 +52,17 @@ void foo() {
 
 struct A {
   A() {
-    clang_analyzer_dump(__func__);
-    clang_analyzer_dump(__FUNCTION__);
-    clang_analyzer_dump(__PRETTY_FUNCTION__);
-    // expected-warning@-3 {{&Element{"A",0 S64b,char}}}
-    // expected-warning@-3 {{&Element{"A",0 S64b,char}}}
-    // expected-warning@-3 {{&Element{"A::A()",0 S64b,char}}}
+#ifdef ANALYZER_MS
+    clang_analyzer_dump(__func__);     // expected-warning {{&Element{"A",0 S64b,char}}}
+#else
+        clang_analyzer_dump(__func__); // expected-warning {{&Element{"A",0 S64b,char}}}
+#endif
```

Comment:
```suggestion
    clang_analyzer_dump(__func__);     // expected-warning {{&Element{"A",0 S64b,char}}}
```
The `#ifdef` and `#else` branch seem identical to me; if that’s not a copy-and-paste error, then I don’t think we need the same check twice.